ultra/tests/test_episode.py

```diff
@@ -36,7 +36,7 @@
 task_id = "00"
 task_level = "easy"
 
-
+# pytype: disable=name-error
```

Comment:
Where is this error coming from? This is either a bug with `pytest` or a bug with the code here.

Reply:
It was coming from `episode` being undefined, this is now fixed